pkgs/development/tools/build-managers/dub/default.nix

```diff
@@ -24,6 +24,15 @@ let
     postPatch = ''
         substituteInPlace test/fetchzip.sh \
             --replace "dub remove" "\"${dubvar}\" remove"
+
+        substituteInPlace test/interactive-remove.sh \
```

Comment:
Maybe you could just apply https://github.com/dlang/dub/commit/2045a9512229cd85d20b89717e3b534c375578fa.diff

Otherwise you can combine these:
```bash
substituteInPlace test/interactive-remove.sh \
  --replace "0.9.20" "1.9.0" \
  --replace "0.9.21" "1.10.0" \
  --replace "select.*0\.9\.20.*0\.9\.21.*" "select.*1\.9\.0.*1\.10\.0.*"
```

Reply:
You are right that would have been better.
Didn't know that multiple replace are allowed.

I must say though that I don't see the necessity to change the commit as this branch is dying off anyway and it's just a cosmetic detail.

Reply:
Oh, we are talking "release-18.09" branch

Reply:
Exactly